tests/test_check_parallel.py

```diff
@@ -28,6 +27,13 @@
 from pylint.typing import FileItem
 from pylint.utils import LinterStats, ModuleStats
 
+try:
+    from concurrent.futures import ProcessPoolExecutor
+    from concurrent.futures.process import BrokenProcessPool
+except ImportError:
+    ProcessPoolExecutor = None  # type: ignore[assignment,misc]
+    BrokenProcessPool = None  # type: ignore[assignment,misc]
```

Comment:
```suggestion
from concurrent.futures import ProcessPoolExecutor
from concurrent.futures.process import BrokenProcessPool
```

Let's make this necessary for tests still.